apps/aecore/lib/aecore/structures/spend_tx.ex

```diff
@@ -28,12 +29,14 @@ defmodule Aecore.Structures.SpendTx do
 
   @spec create(binary(), binary(), non_neg_integer(), non_neg_integer(), non_neg_integer(), non_neg_integer()) :: {:ok, SpendTx.t()}
   def create(from_acc, to_acc, value, nonce, fee, lock_time_block \\ 0) do
-    {:ok, %SpendTx{from_acc: from_acc,
-                  to_acc: to_acc,
-                  value: value,
-                  nonce: nonce,
-                  fee: fee,
-                  lock_time_block: lock_time_block}}
+    tx_data =
```

Comment:
Comming from our previous comments with @rdimitrovbg, this code should be refactored as stated above.